ui/objects/semantic-ui/js/objects/detail/Carousel.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import Slider from "react-slick";
+import { Modal, Image, Button, Grid, Header } from "semantic-ui-react";
+
+export const ImgCarousel = ({ imagesCollection, getCaption }) => {
```

Comment:
Get caption shouldn't be passed as prop, it is a simple utility function that can be just imported